src/data/internals-telemetry-dashboard.yaml

```diff
@@ -0,0 +1,40 @@
+name: Telemetry Dashboards
+summary: Help us with public data at https://telemetry.mozilla.org and some of its hosted dashboards.
+introduction: |
+  ## About the Telemetry Portal
+  
+  The [Telemetry Portal](https://telemetry.mozilla.org/) is a one-stop shop for indexing public and mozilla-specific
+  data dashboards and documentation. There you can explore aggregated Firefox anonymous usage statistics on the
+  Measurement Dashboard, learn what data is collected on which versions in the Probe Dictionary, and more.
+  
+  ## Who Works On the Telemetry Portal?
+  
+  The Firefox Telemetry team, a group of developers spanning timezones UTC+2 to UTC-4, maintain and develop the index,
+  documentation, and some of the tools and dashboards hosted on https://telemetry.mozilla.org. Most of the time we work
+  on data collection modules within Firefox, but we're also responsible for these resources.
+  
+  ## How Do I Get Started?
+  
+  The code for the Telemetry Portal is in its [GitHub repository](https://github.com/mozilla/telemetry-dashboard/).
+  There you can find open issues that need work. Volunteer by commenting on an issue and we'll assign it to you.
```

Comment:
How about adding a link to the mentored-labeled issue search here (e.g. https://github.com/mozilla/telemetry-dashboard/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Amentored)?

This way users could directly jump to the right place.

Reply:
I think it will be displayed in the table, as shown here http://codetribute.netlify.com/projects/internals which show all bugs from telemetry dashboard with mentored label and unassigned.

Reply:
Alright, then the additional link here is not needed.